### PR TITLE
Add parsing of scheduler info to config

### PR DESCRIPTION
**Pull request by msindeeva**
#### What does this implement/fix? Explain your changes.
This adds parsing of scheduler information from the config file for dataset training instead of specifying the LR scheduler in a `TrainModel` class (which is not and should not be logged with each run). **This change does not impact sampler training**

#### What testing did you do to verify the changes in this PR?
Ran training script with new configs

### CODE REVIEW

File: selene_sdk/utils/config_utils.py [hunk 1]
```diff
@@ -321,6 +321,7 @@ def execute(operations, configs, output_dir):
                     == train_loader.dataset.n_target_features
                 )
 
+            # load model, criterion and optimizer
```
Comment (arlapin) [inline selene_sdk/utils/config_utils.py:324]: nit: Oxford comma

**Review by arlapin — APPROVED** [review]
LGTM
